File: Logic_API/src/main/java/dataTransferObjects/WorkGroup.java
package dataTransferObjects;

import java.util.ArrayList;

public class WorkGroup
{
    private User Creator;
    private User ScrumMaster;
    private User ProductOwner;
    private String Name;
    private ArrayList<User> Developers;

    public WorkGroup(String projectname, User creator)    {
        Creator = creator;
        Name = projectname;
    }

    public WorkGroup(String projectname)
    {
        Name = projectname;
    }

    public WorkGroup()    {
    }

    public WorkGroup(User creator, User scrumMaster, User productOwner, String name, ArrayList<User> devs)  {
        Creator = creator;
        ScrumMaster = scrumMaster;
        ProductOwner = productOwner;
        Name = name;
        Developers = devs;
    }

    public User getCreator()
    {
        return Creator;
    }

    public User getScrumMaster()
    {
        return ScrumMaster;
    }

    public User getProductOwner()
    {
        return ProductOwner;
    }

    public String getName()
    {
        return Name;
    }

    public void setCreator(User creator)
    {
        this.Creator = creator;
    }

    public void setScrumMaster(User scrumMaster)
    {
        this.ScrumMaster = scrumMaster;
    }

    public void setProductOwner(User productOwner)
    {
        this.ProductOwner = productOwner;
    }

    public void setName(String name)
    {
        this.Name = name;
    }

    public void addDeveloper(User e)
    {
        Developers.add(e);
    }

    public ArrayList<User> getAllDevelopers()
    {
        return Developers;
    }

    public User getDeveloper(int i)
    {
       return Developers.get(i);
    }
}

File: Data/DatabaseModels/WorkGroup.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Data.DatabaseModels
{
    public class WorkGroup
    {
        [Key]
        public int WorkGroupId { get; set; }
        public User Creator { get; set; }
        public User ScrumMaster { get; set; }
        public User ProductOwner { get; set; }
        [ForeignKey("Project")]
        public string Name { get; set; }
        public List<User> Developers = new List<User>();
        public Project Project { get; set; }
       public WorkGroup(string projectname,User creator)
        {
            Name = projectname;
            Creator = creator;
        }
        public WorkGroup(string projectname)
        {
            Name = projectname;
        }
        public WorkGroup()
        {
        }
    }
}
